app/views/layouts/application.html.haml

```diff
@@ -20,8 +20,7 @@
     = chartbeat_head_block
     = include_mixpanel
 
-    = include_base_css_framework
```

Comment:
Yeah, I failed to read the diff even though I checked twice m( Sorry, ignore that.